docs/cli/edgedb_instance/edgedb_instance_credentials.rst

```diff
@@ -0,0 +1,37 @@
+.. _ref_cli_edgedb_instance_credentials:
+
+
+===========================
+edgedb instance credentials
+===========================
+
+Display instance credentials (add ``--json`` for verbose).
+
+.. cli:synopsis::
+
+     edgedb instance credentials [options] [connection-options]
+
+
+Description
+===========
+
+``edgedb instance credentials`` is a terminal command for displaying the
+credentials of an EdgeDB instance.
+
+
+Options
+=======
+
+:cli:synopsis:`--json`
+    Output in JSON format (password is included in cleartext).
```

Comment:
This completes the change suggested above and adds note of the TLS certs being included in the output here.

```suggestion
    Output in JSON format. In addition to formatting the credentials as JSON,
    this option also includes the password in cleartext and the TLS
    certificates.
```